src/analyze/analyze-security.c

```diff
@@ -1916,14 +1921,15 @@ static int acquire_security_info(sd_bus *bus, const char *name, struct security_
         if (!path)
                 return log_oom();
 
-        r = bus_map_all_properties(bus,
-                                   "org.freedesktop.systemd1",
-                                   path,
-                                   security_map,
-                                   BUS_MAP_STRDUP|BUS_MAP_BOOLEAN_AS_BOOL,
-                                   &error,
-                                   NULL,
-                                   info);
+        r = bus_map_all_properties(
+                bus,
+                "org.freedesktop.systemd1",
+                path,
+                security_map,
+                BUS_MAP_STRDUP | BUS_MAP_BOOLEAN_AS_BOOL,
+                &error,
+                NULL,
+                info);
```

Comment:
I've not followed the previous discussion, but we usually use 16ch indentation for wrapping arguments on calling functions. Though, I do not oppose this change...